<launch>
    <param name="robot_name" value="zoidstein" />
    <param name="armature_name" value="Armature" />
    <param name="joint_axes_yaml" value="$(find ros_bge_bridge)/launch/zoidstein_joint_axes.yaml" />

    <node pkg="ros_pololu_servo" type="ros_pololu_servo_node" name="ros_pololu_servo_node" output="screen">
        <param name="pololu_motors_yaml" value="$(find ros_pololu_servo)/launch/pololu_motors.yaml" />
        <param name="port_name" value="/dev/ttyACM0" />
        <param name="baud_rate" value="115200" />
        <param name="rate_hz" value="10" />
    </node>

    <node pkg="ros_bge_bridge" type="run_bge_controllers.py" name="bge_controllers" output="screen">
        <param name="bge_controllers_yaml" value="$(find ros_bge_bridge)/launch/zoidstein_bge_controllers.yaml" />
    </node>

    <node pkg="ros_bge_bridge" type="launch_blender" name="launch_blender" output="screen">
        <param name="blend_file" value="$(find ros_bge_bridge)/launch/zoidstein.blend" />
        <param name="use_game_engine" value="true" />
    </node>
</launch>
